content/learn/03.programming/05.arduino-and-python/arduino-and-python.md

```diff
@@ -1,17 +1,17 @@
 ---
-author: 'Karl Söderby'
-title: 'MicroPython with Arduino Boards'
-description: 'Learn about compatibility between the popular MicroPython implementation and Arduino boards, how to set up your environment, and discover in-depth tutorials and useful links.'
-tags: 
+author: "Karl Söderby"
```

Comment:
Was this file modified by the linter?

Reply:
Noe i use this file as a test, restored with the main version with the last push